### PR TITLE
Add direct inelastic tests for data manipulation tab

### LINKED ISSUE #35592: The indirect testing docs need updating
The testing docs do not take into account the new inelastic interface. It might be worth making a seperate set of testing docs that use both direct and indirect data. https://developer.mantidproject.org/Testing/IndirectInelastic/IndirectInelasticAcceptanceTests.html

What do you think @SilkeSchomann ?

### PR DESCRIPTION
**Pull request by MohamedAlmaki**
### Description of work
This pull request introduces new manual testing documentation for the inelastic data manipulation tab, which was previously only tested with indirect data. It uses the same instruction used for indirect data.

The PR also does a small bit of re-organised the manual testing directories for direct/indirect/inelastic so that it matches the interface category directory structure in Mantid. The links used in the manual testing issues will be updated in this PR: https://github.com/mantidproject/documents/pull/118

Fixes #35592

### To test:
- Download the new updated sample data from the mantid project website
- Follow the Data Manipulation testing documentation and make sure the instructions are clear

---

### Reviewer

Please comment on the points listed below ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)).
**Your comments will be used as part of the gatekeeper process, so please comment clearly on what you have checked during your review.** If changes are made to the PR during the review process then your final comment will be the most important for gatekeepers. In this comment you should make it clear why any earlier review is still valid, or confirm that all requested changes have been addressed.

#### Code Review

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?
- Do the release notes conform to the [release notes guide](https://developer.mantidproject.org/Standards/ReleaseNotesGuide.html)?

#### Functional Tests

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

### Gatekeeper

If you need to request changes to a PR then please add a comment and set the review status to "Request changes". This will stop the PR from showing up in the list for other gatekeepers.

### CODE REVIEW

**Comment by robertapplin** [comment]
@SilkeSchomann has checked the Direct documentation still looks fine. And I have checked the Data Analysis and Reduction documentation looks good. All that needs testing is the Data Manipulations documentations

File: dev-docs/source/Testing/Inelastic/DataManipulationTests.rst [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+--------------
+
+1. Direct data
+##############
+
+#. Go to ``Interfaces`` > ``Inelastic`` > ``Data Manipulation``
+#. Go to the ``Elwin`` tab
+#. Enter ``MAR27691_red.nxs`` in ``Input file``
+#. Click ``Run`` - this should produce 3 new workspaces ``_elf``, ``_eq`` and ``_eq2``
+#. Now in ``Input file`` choose browse, navigate to the ISIS-Sample data and select the two files, MAR27691_red.nxs and MAR27698_red.nxs, by using shift key
```
Comment (jclarkeSTFC) [inline dev-docs/source/Testing/Inelastic/DataManipulationTests.rst:28]:
```suggestion
#. Now in ``Input file`` choose browse, navigate to the ISIS-Sample data and select the two files, ``MAR27691_red.nxs`` and ``MAR27698_red.nxs``, by using shift key
```

File: dev-docs/source/Testing/Inelastic/DataManipulationTests.rst [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+.. _elwin_indirect_test:
+
+2. Indirect data
+################
+
+#. Go to ``Interfaces`` > ``Inelastic`` > ``Data Manipulation``
+#. Go to the ``Elwin`` tab
+#. Enter ``irs26176_graphite002_red.nxs`` in ``Input file``
+#. Click ``Run`` - this should produce 3 new workspaces ``_elf``, ``_eq`` and ``_eq2``
+#. Now in ``Input file`` choose browse, navigate to the ISIS-Sample data and select the two files above simultaneously, by using shift key
```
Comment (jclarkeSTFC) [inline dev-docs/source/Testing/Inelastic/DataManipulationTests.rst:43]:
```suggestion
#. Now in ``Input file`` choose browse, navigate to the ISIS-Sample data and select ``irs26176_graphite002_red.nxs`` and ``irs26174_graphite002_red.nxs`` simultaneously, by using shift key
```

File: dev-docs/source/Testing/Inelastic/DataManipulationTests.rst [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+
+1. Direct data
+##############
+
+#. Go to ``Interfaces`` > ``Inelastic`` > ``Data Manipulation``
+#. Go to the ``Iqt`` tab
+#. Load the ``MAR27691_sqw.nxs`` file from the sample data
+#. Load the resolution file ``MAR27698_sqw.nxs`` from the sample data
+#. Click ``Run``
+#. A new workspace with the suffix ``_iqt`` should appear in the main GUI, it should be a workspace with 281 histograms and 38 bins.
```
Comment (jclarkeSTFC) [inline dev-docs/source/Testing/Inelastic/DataManipulationTests.rst:66]: My workspace had 17 histograms and 3 bins, still worked fine though.

File: dev-docs/source/Testing/Inelastic/DataManipulationTests.rst [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+**Time required 3 - 5 minutes for each**
+
+--------------
+
+1. Direct data
+##############
+
+#. Go to ``Interfaces`` > ``Inelastic`` > ``Data Manipulation``
+#. Go to the ``Iqt`` tab
+#. Load the ``MAR27691_sqw.nxs`` file from the sample data
```
Comment (jclarkeSTFC) [inline dev-docs/source/Testing/Inelastic/DataManipulationTests.rst:63]:
```suggestion
#. Load the ``MARI27691_sqw.nxs`` file from the sample data
```

File: dev-docs/source/Testing/Inelastic/DataManipulationTests.rst [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+#. Load the resolution file ``MAR27698_sqw.nxs`` from the sample data
```
Comment (jclarkeSTFC) [inline dev-docs/source/Testing/Inelastic/DataManipulationTests.rst:64]:
```suggestion
#. Load the resolution file ``MARI27698_sqw.nxs`` from the sample data
```

File: dev-docs/source/Testing/Inelastic/DataManipulationTests.rst [hunk 6]
```diff
@@ -0,0 +1,84 @@
...
+#. Click ``Run``
+#. A new workspace with the suffix ``_iqt`` should appear in the main GUI, it should be a workspace with 281 histograms and 38 bins.
+#. Click ``Plot Current preview`` this should plot the same data as the preview window
+#. Choose some workspace indices (e.g. 0-2) in the ``Output`` section and click ``Plot Spectra`` this should give a plot with the title *MAR27691_iqt*
```
Comment (jclarkeSTFC) [inline dev-docs/source/Testing/Inelastic/DataManipulationTests.rst:68]:
```suggestion
#. Choose some workspace indices (e.g. 0-2) in the ``Output`` section and click ``Plot Spectra`` this should give a plot with the title *MARI27691_iqt*
```

**Review by jclarkeSTFC — CHANGES_REQUESTED** [review]
Looks good, I only went through the Data Manipulation instructions. I found a couple of small things but everything worked fine when I followed them.

File: dev-docs/source/Testing/Inelastic/DataManipulationTests.rst (hunk 3 above)
Comment (robertapplin) [reply dev-docs/source/Testing/Inelastic/DataManipulationTests.rst]: Thanks for checking, I think this is correct so I'll update the testing docs

**Review by jclarkeSTFC — APPROVED** [review]

**Pull request merged**